src/Netflix/Netflix.ts

```diff
@@ -0,0 +1,760 @@
+// lipsurf-plugins/src/Netflix/Netflix.ts
```

Comment:
Can you put a top-level comment describing any potential "surprises" with this plugin (eg. limitations, things left todo)

Reply:
Will do. Any format I can copy from?